indico/modules/events/abstracts/forms.py

```diff
@@ -212,7 +212,11 @@ class AbstractJudgmentFormBase(IndicoForm):
     accepted_track = QuerySelectField(_("Track"), [HiddenUnless('judgment', AbstractAction.accept)],
                                       get_label='title', allow_blank=True, blank_text=_("Choose a track..."),
                                       description=_("The abstract will be accepted in this track"))
```

Comment:
We should make sure that the description text of the fields is adjusted accordingly for the bulk operations (we are not talking about a single abstract anymore, but many).